x-pack/plugins/watcher/public/sections/watch_edit/components/json_watch_edit/json_watch_edit_simulate_results.tsx

```diff
@@ -110,21 +114,55 @@ export const JsonWatchEditSimulateResults = ({
     },
   ];
 
+  const flyoutTitle = (
+    <EuiTitle size="s">
+      <h2 id="simulateResultsFlyOutTitle">
+        {i18n.translate('xpack.watcher.sections.watchEdit.simulateResults.title', {
+          defaultMessage: 'Simulation results',
+        })}
+      </h2>
+    </EuiTitle>
+  );
+
+  if (error) {
+    return (
+      <EuiFlyout
+        onClose={() => {
+          onCloseFlyout();
+        }}
+        size="s"
+        aria-labelledby="simulateResultsFlyOutTitle"
+      >
+        <EuiFlyoutHeader hasBorder>{flyoutTitle}</EuiFlyoutHeader>
+        <EuiFlyoutBody>
+          <SectionError
+            title={
+              <FormattedMessage
+                id="xpack.watcher.sections.watchEdit.simulateResults.errorTitle"
+                defaultMessage="Cannot simulate watch"
+              />
+            }
+            error={error}
+          />
+        </EuiFlyoutBody>
+      </EuiFlyout>
+    );
+  }
+
+  if (!executeResults) {
+    return null;
+  }
+
   return (
     <EuiFlyout
       onClose={() => {
         onCloseFlyout();
       }}
+      size="s"
```

Comment:
I think we should remove this, since the table loses legibility on narrower screens.

![image](https://user-images.githubusercontent.com/1238659/57480476-aefac480-7254-11e9-802b-c4c293973703.png)

Reply:
fixed